### PR TITLE
Dish, Dish Manager, Dish Controller and JUnit test

### LINKED ISSUE #8: Add Dish

### PR DESCRIPTION
**Pull request by BotBw**
Close #8 #9 

If I didn't get anything wrong.

### CODE REVIEW

File: src/main/java/seedu/duke/controllers/DishController.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+import seedu.duke.entities.Dish;
+import seedu.duke.manager.DishManager;
+
+import java.util.Scanner;
+
+/**
+ * DishController (or MenuController if you like).
+ */
+public class DishController extends Controller {
+    private static final String[] defaultChoice = {"Exit Menu", "Add Dish", "Delete Dish",
```
Comment (darylhjd) [inline src/main/java/seedu/duke/controllers/DishController.java:12]: probably change this to `CHOICES` since it is final.

**Review by darylhjd — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/duke/controllers/DishController.java (hunk 1 above)
Comment (darylhjd) [reply src/main/java/seedu/duke/controllers/DishController.java:12]: the name i mean

Comment (BotBw) [reply src/main/java/seedu/duke/controllers/DishController.java:12]: > the name i mean

Ahh thx

**Review by darylhjd — APPROVED** [review]
ok

**Pull request merged**